rs/bitcoin/kyt/src/lib.rs

```diff
@@ -62,32 +61,9 @@ impl FetchEnv for KytCanisterEnv {
         max_response_bytes: u32,
     ) -> Result<Transaction, HttpGetTxError> {
         // TODO(XC-159): Support multiple providers
-        let host = "btcscan.org";
-        let url = format!("https://{}/api/tx/{}/raw", host, txid);
-        let request_headers = vec![
-            HttpHeader {
-                name: "Host".to_string(),
-                value: format!("{host}:443"),
-            },
-            HttpHeader {
-                name: "User-Agent".to_string(),
-                value: "bitcoin_inputs_collector".to_string(),
-            },
-        ];
-        let request = CanisterHttpRequestArgument {
-            url: url.to_string(),
-            method: HttpMethod::GET,
-            body: None,
-            max_response_bytes: Some(max_response_bytes as u64),
-            transform: Some(TransformContext {
-                function: TransformFunc(candid::Func {
-                    principal: ic_cdk::api::id(),
-                    method: "transform".to_string(),
-                }),
-                context: vec![],
-            }),
-            headers: request_headers,
-        };
+        let request =
+            providers::create_request(state::get_config().network, txid, max_response_bytes);
```

Comment:
nit: unnecessary path prefix due to the import
```suggestion
            providers::create_request(get_config().network, txid, max_response_bytes);
```